docs/pages/careers/support-agent.md

```diff
@@ -37,49 +37,63 @@ For additional details about the culture, you can check our [careers](https://mu
+- Create Notion pages to suggest opportunities to improve the quality and efficiency of our customer service operation overall
 
-- Respond to customer inquiries via email in a clear, concise, and comprehensive manner.
-  You might provide refunds, answer license questions, process purchase-orders, etc.
-- Assist in writing and maintaining the FAQs and guides.
-- Suggest opportunities to make customers happier for our product team.
-- Suggest opportunities to improve the quality and efficiency of our customer service operation.
 
 ## Who we're looking for
 
 ### Required
 
-- 1+ years' experience in a Customer Support role.
-- Exceptional writing skills (be clear and concise).
-- Experience working remotely.
+- **Independent learning skills:** we operate mainly on documentation training and feedback, so having someone who can learn by reading through Notion pages, previous tickets, and written instructions and then responding to async feedback is key.
+- **Excellent communication skills:** the bulk of the work will be interpreting and responding to customer inquiries, so clear writing skills are invaluable.
+- **Patience and curiosity:** support requests can be tedious and confusing sometimes! We need someone who knows how to follow-up for clarification, and who is committed to resolving requests with kindness and knowledgeability.
+- **Organised thinking:** responding to tickets requires an ability to interpret an incoming ticket based on existing or new patterns (agreement terms, support types, product promises, etc) and to match requests with the appropriate responses (from macros, previous tickets, or templates).
+- **Self-management:** as a remote and async company, we firmly avoid micromanaging practices, relying on everyone to leverage their own strategies for productivity and focusing on output, not process.
+- **Organised documenter:** we thrive when using thorough documentation and categorisation, meaning tickets are labeled, patterns become templates, and ideas for automation/improvement become entries in our project database.
+- **Comfortable around technical language:** engineering skills are absolutely not required, however you will need some basic literacy around web development and digital products work (downloads, installs, upgrades, updates, bugs, etc).
+
 
 ### Nice to have (but not required)
 
-- Basic knowledge of web development.
+- Experience (1+ years) providing customer support for similar companies (developer tools, IT tools, digital products) will be considered a bonus.
 
 ## Benefits and compensation
 
-Competitive compensation depending on the profile and location of up to $20/h.
-You can find the other perks & benefits on the [careers](https://mui.com/careers/#perks-and-benefits) page.
+**Compensation bands:** $20-30k USD or equivalent in local currency.
```

Comment:
```suggestion
**Compensation bands:** 20-30k USD, or equivalent in local currency, per year.
```

For the sake of extra clarity, it may be worth clarifying that this is (I assume) a yearly-based comp.